packages/test/src/workflows/smorgasbord.ts

```diff
@@ -17,17 +17,10 @@ import * as activities from '../activities/';
 import { signalTarget } from './signal-target';
 import { activityStartedSignal, unblockSignal } from './definitions';
 
-const { fakeProgress } = proxyActivities<typeof activities>({
-  startToCloseTimeout: '5s',
-  scheduleToCloseTimeout: '10s',
-  heartbeatTimeout: '3s',
+const { fakeProgress, queryOwnWf } = proxyActivities<typeof activities>({
+  startToCloseTimeout: '1m',
```

Comment:
Doesn't this still need a heartbeat timeout?

Reply:
Not anymore since we have the default on the worker side now.
Adding the timeout just increases flakiness.